psh/pshapp/pshapp.c

```diff
@@ -521,6 +534,26 @@ static int psh_keyCode(const char *buff, int *pEsc)
 }
 
 
+static int psh_skipword(const char *str, int n, int iswordchr(int))
```

Comment:
I don't prefer to use variable names like `n` - it's much easier to read a code with longer ones - but in this case I don't have a better idea and I'd leave it - implementation is much simpler thanks to the solution applied.